invenio_rdm_records/services/schemas/__init__.py

```diff
@@ -51,22 +47,13 @@ class Meta:
         'files': 'read_files',
     }
 
-    # schema_version = fields.Interger(dump_only=True)
-    # revision = fields.Integer(attribute='revision_id', dump_only=True)
-    # id = fields.Str(attribute='recid', dump_only=True)
-    # concept_id = fields.Str(attribute='conceptrecid', dump_only=True)
-    created = fields.Str(dump_only=True)
-    updated = fields.Str(dump_only=True)
-
-    # status = fields.Str(dump_only=True)
-
+    id = fields.Str()
+    conceptid = fields.Str()
     metadata = NestedAttribute(MetadataSchemaV1)
     access = NestedAttribute(AccessSchemaV1)
-    # files = NestedAttribute(FilesSchemaV1, dump_only=True)
-    # communities = NestedAttribute(CommunitiesSchemaV1)
-    # pids = NestedAttribute(PIDSSchemaV1)
-    # stats = NestedAttribute(StatsSchemaV1, dump_only=True)
-    # relations = NestedAttribute(RelationsSchemaV1, dump_only=True)
+    pids = fields.List(NestedAttribute(PIDSchemaV1))
```

Comment:
Comment: Let's drop the ``V1`` prefix on everything and only add it once we need it.

Reply:
Done to the `schemas` folder, `marshmallow` should be removed. I will work on that now.